src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const oneAction of actions) {
```

Comment:
что-то в единственном числе предполагает что это будет одно
```suggestion
  for (const action of actions) {
```

Reply:
Mені так зручніше писати, бо -s мoжна не помітити